experiments/modern/src/runtime/Group.js

```diff
@@ -10,10 +10,11 @@ class Group extends GuiObject {
   static getClassName() {
     return "Group";
   }
-  findObject(id) {
-    throw new Error("Not implemented");
+  getObject(id) {
+    // Not sure this is correct, but it is my understanding this is just an alias
+    return this.findObject(id);
```

Comment:
Maybe this one only searches direct children, rather than doing a deep search?

Reply:
oooh, that would be interesting. Think I will leave it for now with comment, and we can do further testing to decipher it later.